docs/reference/query-builders.md

```diff
@@ -0,0 +1,382 @@
+
+### `defaultSort()`
+
+```
+query.defaultSort({ updatedAt: 1 })
+```
+
+The `defaultSort` changes the default value for the [`sort` query builder](#sort).  The argument is the same as for the `sort` query builder: an object like `{ updatedAt: 1 }`, including a field name and `1` for ascending or `-1` for descending. See the [MongoDB `sort` method documentation](https://docs.mongodb.com/manual/reference/method/cursor.sort/) for other options. `false` can be passed as an option as well to clear the default sort.
+
+It is distinct from the `sort` builder to distinguish between cases where a default sort should be ignored (for instance, the `search` query builder is present) and cases where a sort is explicitly requested by the user.
+
+In most custom uses the `sort` builder will likely be a more common option.
+
+<!--
+### `distinctCounts()`
+TODO
+```
+query.distinctCounts(true)
+```
+
+`.distinctCounts(true)` makes it possible to obtain counts for each distinct value after a call to `toCount()` is resolved by calling `query.get('distinctCounts')`. These are returned as an object in which the keys are the distinct values of the field, and the values are the number of occurrences for each value.
+
+This has a performance impact.
+-->
+
+### `_ids()`
+
+```
+query._ids([
+  'ckwxzxdtr000t4v3rivysoari:en:published',
+  'ckwxzxdvc001b4v3rhxvxqgg1:en:published'
+])
+```
+
+The `_ids` builder causes the query to return only the document with matching `_id` properties and to return them in that order, assuming the documents with the specified IDs exist. All documents are fetched in the same locale regardless of the locale suffix of the values. If no locale can be determined via query parameters, the locale is inferred from the first _id in the set.
+
+The query builder can also be called with a string, which is treated as a single document ID.
+
+### `limit()`
+
+```
+query.limit(10)
+```
+
+The `limit` query builder accepts an integer and limits the number of document results. It returns the first matching documents up to the set limit, though it does take the [`skip` builder](#skip) into account.
+
+### `locale()`
+
+```
+query.locale('es:published')
+```
+
+A valid locale identifier passed to the `locale` builder will tell the query to return results only belonging to that locale. Documents that match the query and have *no locale* (because their doc type is not localized) are also included in the results. Specifically passing `false` (the default value) will use the locale and mode on the `req` request object.
```

Comment:
"to that locale, including the mode (draft or published) after a `:` character."